### PR TITLE
Fix email notification url

### PR DESCRIPTION
**Pull request by marcinbujar**

### CODE REVIEW

**Review by jeho2a — APPROVED** [review]
Can't see this could be a problem: fixes the bug I reported a while ago if I recall.

**Comment by marcinbujar** [comment]
> Can't see this could be a problem: fixes the bug I reported a while ago if I recall.

Thanks Jim. Looks like you don't have write access and hence the PR can't be merged after your approval. @stevejuma ?

**Pull request merged**
